packages/server/src/arpa_reporter/services/validate-upload.js

```diff
@@ -349,6 +349,19 @@ async function validateRecord({ upload, record, typeRules: rules }) {
                 }
             }
 
+            if (rule.dataType === 'Numeric') {
+                console.log
```

Comment:
was this `console.log` committed by mistake here?

Reply:
ah yes, that must've slipped through while I was trying to debug something. Removed now, thanks!